c2rust-refactor/src/scripting.rs

```diff
@@ -511,6 +540,33 @@ impl<'a, 'tcx> TransformCtxt<'a, 'tcx> {
 #[allow(unused_doc_comments)]
 impl<'a, 'tcx> UserData for TransformCtxt<'a, 'tcx> {
     fn add_methods<'lua, M: UserDataMethods<'lua, Self>>(methods: &mut M) {
+        methods.add_method(
+            "dump_marks",
+            |_lua_ctx, this, ()| Ok(println!("Marks: {:?}", this.st.marks())),
+        );
+
+        methods.add_method_mut(
+            "clear_marks",
+            |_lua_ctx, this, ()| Ok(this.st.marks_mut().clear()),
+        );
+
+        methods.add_method(
+            "get_marks",
+            |lua_ctx, this, ()| {
```

Comment:
Is there any way to not duplicate this and related functionality in `RefactorState` and `TransformCtxt`?

Reply:
Not sure I follow - what functionality is this duplicating? All this function does is serialize the marks to lua

Reply:
Just above that there's a `get_marks` method on `RefactorState` that is identical.

Reply:
Oh lol. Not sure how that happened. I definitely think we don't need both